GiEnJul/ClientApp/src/components/admin/AdminPage.tsx

```diff
@@ -1,23 +1,89 @@
-import { withAuthenticationRequired } from "@auth0/auth0-react";
+import { useAuth0, withAuthenticationRequired } from "@auth0/auth0-react";
 import { Container } from "@material-ui/core";
 import * as React from "react";
+import { useEffect, useState } from "react";
 import LoadingPage from "../../common/components/LoadingPage";
+import InstitutionMacro from "../institution/InstitutionMacro";
 import LogOutButton from "../login/LogOutButton";
 import AdminMenu from "./common/AdminMenu";
 import Completed from "./connections/Completed";
 import Suggested from "./connections/Suggested";
 
+type UserDataType = {
```

Comment:
Should all variables be nullable? Or should we require certain variables from the 
`getUserInformation().then((response: UserDataType) => setUserData(response)),` to actually be set?

Reply:
Depends on what variables we really need